### PR TITLE
remove tensorrt installation

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##

Don't install tensorrt

### CODE REVIEW

**Comment by rohithkrn** [comment]
are we making this change due to license concerns?

File: serving/docker/scripts/install_tensorrt.sh [hunk 1]
```diff
@@ -1,47 +0,0 @@
-#!/bin/bash
```
Comment (rohithkrn) [inline serving/docker/scripts/install_tensorrt.sh:1]: i am leveraging this script in the internal build, should we remove that too?

File: serving/docker/tensorrt-llm.Dockerfile [hunk 2]
```diff
@@ -9,12 +9,15 @@
 # or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS"
 # BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
 # the specific language governing permissions and limitations under the License.
-ARG version=12.2.2-runtime-ubuntu22.04
+ARG version=12.2.2-cudnn8-runtime-ubuntu22.04
```
Comment (rohithkrn) [inline serving/docker/tensorrt-llm.Dockerfile:12]: does it have tensorrt pre-installed?

File: serving/docker/scripts/install_tensorrt.sh (hunk 1 above)
Comment (lanking520) [reply serving/docker/scripts/install_tensorrt.sh:1]: no, internal build you still need trt

File: serving/docker/tensorrt-llm.Dockerfile (hunk 2 above)
Comment (lanking520) [reply serving/docker/tensorrt-llm.Dockerfile:12]: no

File: serving/docker/scripts/install_tensorrt.sh (hunk 1 above)
Comment (rohithkrn) [reply serving/docker/scripts/install_tensorrt.sh:1]: i meant if we are okay to use it from legal point of view

File: serving/docker/tensorrt-llm.Dockerfile (hunk 2 above)
Comment (rohithkrn) [reply serving/docker/tensorrt-llm.Dockerfile:12]: curious why are we switching from runtime image?

File: serving/docker/scripts/install_tensorrt.sh (hunk 1 above)
Comment (lanking520) [reply serving/docker/scripts/install_tensorrt.sh:1]: to build it is ok, but we have some issues with libnvinfer.so

File: serving/docker/tensorrt-llm.Dockerfile (hunk 2 above)
Comment (lanking520) [reply serving/docker/tensorrt-llm.Dockerfile:12]: this is a clean install and save 2 GB of memory. Now the entire container takes 11GB uncompressed

Comment (rohithkrn) [reply serving/docker/tensorrt-llm.Dockerfile:12]: cudnn image takes less space than runtime image?

**Review by rohithkrn — APPROVED** [review]

File: serving/docker/tensorrt-llm.Dockerfile (hunk 2 above)
Comment (lanking520) [reply serving/docker/tensorrt-llm.Dockerfile:12]: takes more, but we anyway install everything manually, so it is the same

Comment (lanking520) [reply serving/docker/tensorrt-llm.Dockerfile:12]: during installation of tensorrt, we install cudnn8

**Pull request merged**